code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -112,6 +112,27 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	resistance_flags = NONE
 
+/obj/item/clothing/head/syndicatefake/nuke
+	name = "Nuclear Operative Helmet replica"
+	desc = "The feared suit of a Syndicate Nuclear agent, now in plastic!"
+	icon_state = "fake_nuke_helmet"
+	item_state = "fake_nuke_helmet"
+	body_parts_covered = HEAD
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+
+/obj/item/clothing/suit/syndicatefake/nuke
+	name = "Nuclear Operative Hardsuit replica"
+	desc = "The feared suit of a Syndicate Nuclear agent, now in plastic!"
+	icon_state = "fake_nuke_suit"
+	item_state = "fake_nuke_suit"
+	w_class = WEIGHT_CLASS_NORMAL
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman, /obj/item/toy)

```

Comment:
This line needs to be removed, since it's inherited from /obj/item/clothing/suit/syndicatefake